test/runtests.jl

```diff
@@ -21,6 +21,53 @@ function draw2(μ)
     return (x,y)
 end
 
+measures = [
+    # Chain(x -> Normal(μ=x), Normal(μ=0.0))
+    For(3) do j Normal(σ=j) end
+    For(2,3) do i,j Normal(i,j) end
+    # Likelihood
+    # Pointwise
+    Normal() ^ 3
+    Normal() ^ (2,3)
+    # SpikeMixture(Normal(), 2)
+    # Dirac(0.0) + Normal()
+    # transforms
+    3 * Normal()
+    Bernoulli(0.2)
+    Beta(2,3)
+    Binomial(10,0.3)
+    Cauchy()
+    Dirichlet(ones(3))
+    Exponential()
+    Gumbel()
+    # InverseGamma(2)
```

Comment:
Are these commented out because they do not currently work? If so, perhaps better to put them in a different list and then apply `@test_broken`, so that the test suite tells us when they work and we need to move them from one list to the other?

Reply:
Yes, I agree these should be `@test_broken` unless fixed.